core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -276,8 +322,8 @@ oppia.factory('TopicUpdateService', [
        * uncategorized skill ids and records the change in
        * the undo/redo service.
        */
-      moveSkillIdToSubtopic: function(
-          topic, oldSubtopicId, newSubtopicId, skillId) {
+      moveSkillToSubtopic: function(
```

Comment:
Skill or SkillSummary? Update variable names below as well.

Reply:
Even though, on the surface, we are moving just the skill summary to a subtopic. Actually, that entire skill is now under the new subtopic, right? So, I thought moveSkillToSubtopic might be more suited, or should it be moveSkillSummaryToSubtopic because topic object only accesses a skill's summary?

Reply:
Oh I see, that's a good point. I could go either way on this one. You decide.

I think anything that refers to the UI component should be "skillSummary", and anything that refers to the conceptual skill should be "skill".

Reply:
Ok, so in the directives, I will change it to skillSummary and keep it skill here.

Reply:
OK (for keeping "skill" in the method name). But I think some of the internal arg/variable names in this method would still need to be changed.

Reply:
Done.